3001-4000/LIT3183Commen.xml

```diff
@@ -65,6 +58,55 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                <relation name="saws:isCommentOn" active="LIT3183Commen" passive="LIT1315Dodeka"/>
             </listRelation>
          </div>
+         <div type="edition">
+            
+            <div type="textpart" subtype="section" xml:lang="gez" xml:id="Hosea">
+               <label>On Genesis</label>
```

Comment:
Genesis or Hosea? :) Also below, a few textparts have different names in the xml:id and labels

Reply:
Corrected.